docs/modules/api/pages/index.adoc

```diff
@@ -40,10 +40,148 @@ Other languages coming soon
 
 == Request and response formats
 
+[#check-resources]
+=== `CheckResources` (`/api/check/resources`)
+
+This is the main API entrypoint for checking permissions for a set of resources. 
+
+.Request
+[source,json,linenums]
+----
+{
+  "requestId": "test", <1>
+  "principal": { 
+    "id": "alice", <2>
+    "policyVersion": "20210210", <3>
+    "scope": "acme.corp", <4>
+    "roles": [ <5>
+      "employee"
+    ],
+    "attr": { <6>
+      "department": "accounting",
+      "geography": "GB",
+      "team": "design"
+    }
+  },
+  "resources": [ <7>
+    {
+      "resource": { 
+        "id": "XX125", <8> 
+        "kind": "leave_request", <9>
+        "policyVersion": "20210210", <10>
+        "scope": "acme.corp", <11>
+        "attr": { <12>
+          "department": "accounting",
+          "geography": "GB",
+          "id": "XX125",
+          "owner": "john",
+          "team": "design"
+        }
+      },
+      "actions": [ <13> 
+        "view:public",
+        "approve",
+        "create"
+      ]
+    }
+  ],
+  "auxData": { <14> 
+    "jwt": {
+        "token": "xxx.yyy.zzz", <15> 
+        "keySetId": "ks1" <16>
+    }
+  },
+  "includeMeta": true <17>
+}
+----
+<1> Request ID is an optional, application-provided identifier useful for correlating logs.
+<2> ID of the principal whose permissions are being checked. This usually comes from the identity provider (IdP).
+<3> Principal policy version. Optional. The server falls back to the xref:configuration:engine.adoc[configured default version] if this is not specified. 
+<4> Principal policy scope. Optional. See xref:policies:scoped_policies.adoc[].
+<5> The roles attached to this principal by the identity provider.
+<6> Free-form context data about this principal. Policy rule conditions are evaluated based on these values.
+<7> List of resources the principal is attempting to access.
+<8> ID of the resource.
+<9> Resource kind. This is used to determine the resource policy that applies to this resource.
+<10> Resource policy version. Optional. The server falls back to the xref:configuration:engine.adoc[configured default version] if this is not specified. 
+<11> Resource policy scope. Optional. See xref:policies:scoped_policies.adoc[].
+<12> Free-form context data about this resource. Policy rule conditions are evaluated based on these values.
+<13> List of actions being performed on the resource.
+<14> Optional section for providing auxiliary data.
+<15> JWT to use as an auxiliary data source. 
+<16> ID of the keyset to use to verify the JWT. Optional if only a single xref:configuration:auxdata.adoc[keyset is configured].
+<17> Optional flag to receive metadata about request evaluation.
+
+
+.Response
+[source,json,linenums]
+----
+{
+  "requestId": "test", <1>
+  "results": [ <2>
+    {
+      "resource": { <3>
+        "id": "XX125",
+        "kind": "leave_request",
+        "policyVersion": "20210210"
```

Comment:
Should this have `"scope": "acme.corp"` since that was in the request?

Reply:
Good catch!